config.md

```diff
@@ -208,6 +208,7 @@ For Linux-based systems, the `process` object supports the following process-spe
     For more information on how these two settings work together, see [the memory cgroup documentation section 10. OOM Contol][cgroup-v1-memory_2].
 * **`selinuxLabel`** (string, OPTIONAL) specifies the SELinux label for the process.
     For more information about SELinux, see  [SELinux documentation][selinux].
+* **`allowSpeculation`** (bool, OPTIONAL) setting `allowSpeculation` to true disable speculative execution mitigations to improve the performance.
```

Comment:
I think we should name this `disableSpeculationMitigations` instead.

Reply:
>I think we should name this disableSpeculationMitigations instead.

`disableSpeculationMitigations` is easy to understand although I wanted to use the positive boolean variable name. I don't mind whichever it is.
@giuseppe, Could you give me your opinion on the variable name?

Reply:
I think the name @mrunalp suggested is clearer, since it is about allowing or disabling mitigations not the speculative execution itself.

Reply:
Thanks! I'll modify it.

Reply:
@mrunalp @giuseppe I changed the name.